### PR TITLE
Issue #83: Renamed files and re-structured server files

### LINKED ISSUE #83: Organize Project for end of Sprint 4
Must restructure files, especially by cleaning up index.js and separating the routes into their own file. CamelCase syntax must be used, and code must be commented as well. No excess commented blocks should remain.

### PR DESCRIPTION
**Pull request by gregtuc**
All photo routes were moved into the routes package and out of index.js, leaving only essential items. Items that didn't abide by CamelCase conventions were renamed. Files were renamed to lowercase. Unnecessary code-blocks removed. Statements causing warnings during compilation were removed.

### CODE REVIEW

File: client/src/components/displayHomepage.js [hunk 1]
```diff
@@ -2,108 +2,108 @@ import React, { Component } from "react";
 import axios from "axios";
 import { withRouter } from "react-router-dom";
 import {
```
Comment (loganmcmullen) [inline client/src/components/displayHomepage.js:4]: Great improvements of little subtleties.

File: client/src/components/displayPictures.js [hunk 2]
```diff
@@ -44,36 +42,34 @@ class RenderPictures extends Component {
...
         console.log(error);
-      });     
-      setTimeout(()=>{alert("Image Deleted"); window.location.reload(true);}, 2000);
-      //window.location.reload(true);
-
-    }
-
-
+      });
+    setTimeout(() => {
```
Comment (loganmcmullen) [inline client/src/components/displayPictures.js:67]: respect

File: server/routes/registerUser.js [hunk 3]
```diff
@@ -24,15 +24,15 @@ router.post("/", async (req, res) => {
     user = new User({
       email,
```
Comment (loganmcmullen) [inline server/routes/registerUser.js:25]: very, very impressive cleanup in quick time. I approve brother

**Review by loganmcmullen — APPROVED** [review]

File: server/routes/photos.js [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+      const readstream = gfs.createReadStream(file.filename);
+      readstream.pipe(res);
+    } else {
+      res.status(404).json({
+        err: "Not an image",
+      });
+    }
+  });
+});
+module.exports = router;
```
Comment (cottonswab) [inline server/routes/photos.js:255]: Good idea merging all these files into one.

File: server/index.js [hunk 5]
```diff
@@ -86,195 +44,4 @@ app.get("/", (req, res) => {
...
-app.delete("/files/:_id", (req, res) => {
-  gfs.remove({ _id: req.params._id, root: "uploads" }, (err, gridStore) => {
-    if (err) {
-      return res.status(404).json({ err: err });
-    }
-  });
-  res.redirect('/');
-});
-
 module.exports = server;
```
Comment (cottonswab) [inline server/index.js:47]: Good work cutting out unnecessary lines of code.

**Review by cottonswab — APPROVED** [review]
Good work, drastically improves code readability. Approved.

**Pull request merged**